### PR TITLE
Adding a checkbox to the settings page to allow pointing the extension to locahost:8080

### PR DESCRIPTION
**Pull request by nahnhnahk**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -5,6 +5,16 @@ function parseHanVietResponse(responseText) {
     return hanviet;
 }
 
+BACKEND = "https://vocab.nganhan.xyz";
```
Comment (tmkhoa) [inline background.js:8]: const BACKEND?

Comment (tmkhoa) [inline background.js]: let BACKEND = ''

File: options.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Saves options to chrome.storage
+function save_options() {
```
Comment (quynhnga-nguyen) [inline options.js]: change this to camel case

File: options.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  var useLocalhost = document.getElementById('use-localhost').checked;
+  chrome.storage.sync.set({
+    useLocalhost: useLocalhost,
+  }, function() {
+    // Update status to let user know options were saved.
+    var status = document.getElementById('status');
+    status.textContent = 'Options saved.';
+    setTimeout(function() {
+      status.textContent = '';
+    }, 750);
```
Comment (tmkhoa) [inline options.js]: refactor this to be a function?

File: options.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  });
+}
+
+// Restores select box and checkbox state using the preferences
+// stored in chrome.storage.
+function restore_options() {
+  chrome.storage.sync.get({
+    useLocalhost: false,
+  }, function(items) {
+    console.log(items);
```
Comment (tmkhoa) [inline options.js]: is it safe to log user data?

File: background.js (hunk 1 above)
Comment (tmkhoa) [reply background.js]: use double quotes vs single quotes ("" vs '')

File: options.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Debugging options</title>
+    <script type="text/javascript" src="options.js"></script>
+  </head>
+  <body>
+    <label><input type="checkbox" id="use-localhost">Use localhost</label>
```
Comment (quynhnga-nguyen) [inline options.html]: here as well

**Review by quynhnga-nguyen — CHANGES_REQUESTED** [review]

File: options.js (hunk 3 above)
Comment (nahnhnahk) [reply options.js]: Done.

File: options.js (hunk 4 above)
Comment (nahnhnahk) [reply options.js]: Done.

File: options.js (hunk 2 above)
Comment (nahnhnahk) [reply options.js]: Done.

File: options.html (hunk 5 above)
Comment (nahnhnahk) [reply options.html]: Done.

File: background.js (hunk 1 above)
Comment (nahnhnahk) [reply background.js]: Most of the code base is using double quotes, so sticking to it and change all other places in this PR.

**Comment by nahnhnahk** [comment]
Addressing comments in previous iteration.

**Review by quynhnga-nguyen — APPROVED** [review]

**Pull request merged**